src/main/resources/db/migration/all/20240311103635__make_ap_type_column_on_approved_premises_applications_table.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE approved_premises_applications ADD COLUMN ap_type TEXT NOT NULL DEFAULT 'NORMAL';
+
+UPDATE approved_premises_applications
+SET ap_type = 'PIPE'
+WHERE is_pipe_application = true;
+
+UPDATE approved_premises_applications
+SET ap_type = 'ESAP'
+WHERE is_esap_application = true;
+
+ALTER TABLE approved_premises_applications ALTER COLUMN ap_type DROP DEFAULT;
+ALTER TABLE approved_premises_applications DROP COLUMN is_pipe_application;
+ALTER TABLE approved_premises_applications DROP COLUMN is_esap_application;
```

Comment:
This is probably me being paranoid but i wonder if it's worth keeping this around for a few weeks just in case we've missed something re: backfill? I'm probably worrying too much though!

Reply:
This is its own separate migration now.